plugins/woocommerce/src/Blocks/InteractivityComponents/Dropdown.php

```diff
@@ -34,25 +32,17 @@ public static function render( $props ) {
 			'selectType'    => $select_type,
 		);
 
-		$action = $props['action'] ?? '';
-
-		$dropdown_context = array(
-			'selectedItems' => $selected_items,
-			'isOpen'        => false,
-			'selectType'    => $select_type,
-		);
-
-		$action = $props['action'] ?? '';
-
-		$namespace = wp_json_encode( array( 'namespace' => 'woocommerce/interactivity-dropdown' ) );
-
+		$action        = $props['action'] ?? '';
+		$namespace     = wp_json_encode( array( 'namespace' => 'woocommerce/interactivity-dropdown' ) );
 		$wrapper_class = 'multiple' === $select_type ? '' : 'single-selection';
+		$input_id      = wp_unique_id( 'wc-interactivity-dropdown-input-' );
 
-		$input_id = wp_unique_id( 'wc-interactivity-dropdown-input-' );
-
-		$wrapper_class = 'multiple' === $select_type ? '' : 'single-selection';
-
-		$input_id = wp_unique_id( 'wc-interactivity-dropdown-input-' );
+		wp_add_inline_style(
+			'wc-interactivity-dropdown',
+			"#$input_id::placeholder {
+					$text_color_style
+			}"
+		);
```

Comment:
I didn't spot this on the first review, but in my test, we can style the placeholder using CSS only:

```
#wc-interactivity-dropdown-input-3,
#wc-interactivity-dropdown-input-3::placeholder {
	color: inherit;
}
```

Reply:
Oh good point. I didnt think of that at the time. I will fix this.